staging/src/k8s.io/api/README.md

```diff
@@ -1 +1,19 @@
-This repo is still in the experimental stage. Shortly it will contain the schema of the API that are served by the Kubernetes apiserver.
+# api
+
+Schema of the external API types that are served by the Kubernetes API server.
+
+## Purpose
+
+This library is the canonical location of the Kubernetes API definition. Most likely interaction with this repository is as a dependency of client-go.
+
+## Compatibility
+
+As a depenency of client-go, we provide the compatibility guarantees outlined at https://github.com/kubernetes/client-go#versioning, for the respective revisions of this repo in client-go's Godeps.json. It is in direct support of Kubernetes, so branches will track Kubernetes and be compatible with that repo. As we more cleanly separate the layers, we will review the compatibility guarantee.
+
+## Where does it come from?
+
+`api` is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api. Code changes are made in that location, merged into `k8s.io/kubernetes` and later synced here.
+
+## Things you should *NOT* do
+
+1. Directly modify any files under `pkg` in this repo. Those are driven from `k8s.io/kubernetes/staging/src/k8s.io/api`.
```

Comment:
actually none of the files in that repo.

Reply:
> `api` is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api. Code changes are made in that location, merged into `k8s.io/kubernetes` and later synced here.

This line ends up conveying that one shouldn't modify _any_ files under this repo. So we probably don't need this section.

Or if it's better to be explicit: Directly modify any files ~~under `pkg`~~ in this repo

Reply:
Get rid of the relativity:  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api is synched to k8s.io/api. All changes must be made in the former. The later is read-only.

Reply:
I have copy pasted this from https://github.com/kubernetes/apimachinery So maybe should be changed there as well ;)

Done! Sorry for taking so long, was a bit busy as I do this in my personal time.

Reply:
> So maybe should be changed there as well ;)

Yup, looks like it should be.

> Sorry for taking so long, was a bit busy as I do this in my personal time.

That's more than ok, it's great to see the contribution at the end of the day :D